print-802_11.c

```diff
@@ -927,27 +927,30 @@ static const char *reason_text[] = {
 	  "responding station",					/* 9 */
 	"Disassociated because the information in the Power Capability "
 	  "element is unacceptable",				/* 10 */
-	"Disassociated because the information in the SupportedChannels "
+	"Disassociated because the information in the Supported Channels "
 	  "element is unacceptable",				/* 11 */
-	"Invalid Information Element",				/* 12 */
-	"Reserved",						/* 13 */
+	"Disassociated due to BSS transition management",	/* 12 */
+	"Invalid Information Element",				/* 13 */
 	"Michael MIC failure",					/* 14 */
 	"4-Way Handshake timeout",				/* 15 */
 	"Group key update timeout",				/* 16 */
 	"Information element in 4-Way Handshake different from (Re)Association"
 	  "Request/Probe Response/Beacon",			/* 17 */
 	"Group Cipher is not valid",				/* 18 */
+	"Pairwise Cipher is not valid",				/* 19 */
```

Comment:
Should 19 read, "Invalid pairwise cipher?"
![reason_codes](https://user-images.githubusercontent.com/7466510/51317150-6eb0d580-1a1c-11e9-9c57-0bca84606a8c.png)

Reply:
Yes, but I followed how some others were written. If you notice, a few others should be 'invalid' and are written as 'not valid'. Same goes for 'STA', but are written as 'station'.

I can redo this PR and fix all of that if you'd like.

Reply:
I will defer to the maintainers on if a new PR is needed, but I wanted to point it out.  Thank you for the follow up.

Reply:
Any update from the maintainers?

Reply:
Reached any decision on how you want the PR?

Reply:
@guyharris and @Mister-X- was there any next steps for the PR?

Reply:
There isn't anything in the comments above that tell me what the next step on my side would be. I can see a two possibilities on your side: merge it, or close it (and provide with the changes to do